### PR TITLE
[cli][client] Throw an error if both `vercel.json` and `now.json` exist

### PR DESCRIPTION
**Pull request by TooTallNate**

### CODE REVIEW

File: packages/now-client/src/create-deployment.ts [hunk 1]
```diff
@@ -114,10 +114,24 @@ export default function buildCreateDeployment(version: number) {
...
+      const relativePaths = fileList.map(f => relative(cwd, f));
+      const hasVercelConfig = relativePaths.includes('vercel.json');
+      const hasNowConfig = relativePaths.includes('now.json');
+
+      if (hasVercelConfig) {
+        if (hasNowConfig) {
+          throw new DeploymentError({
+            code: 'conflicting_config',
+            message:
+              'Both "vercel.json" and "now.json" exist. Please delete the "now.json" file',
```
Comment (styfle) [inline packages/now-client/src/create-deployment.ts]:
```suggestion
              'Cannot use both a `vercel.json` and `now.json` file. Please delete the `now.json` file.',
```

Let's make this consistent with the ignore error message.

**Review by styfle — DISMISSED** [review]
Let's add a test too?

**Comment by styfle** [comment]
@TooTallNate This doesn't work because the CLI passes the `nowConfig` property so the code from this PR is never executed.

File: packages/now-client/src/create-deployment.ts [hunk 2]
```diff
@@ -114,10 +114,24 @@ export default function buildCreateDeployment(version: number) {
     let configPath: string | undefined;
     if (!nowConfig) {
```
Comment (styfle) [inline packages/now-client/src/create-deployment.ts:86]: `nowConfig` is always an object passed from the CLI so this code is never executed

**Comment by TooTallNate** [comment]
@styfle Added a test to CLI.

**Review by styfle — DISMISSED** [review]

**Review by styfle — COMMENTED** [review]
There's at least one more place that should probably be updated: https://github.com/vercel/vercel/blob/9d73091d8cd69787de8c8add84668b28c466029a/packages/now-cli/src/util/config/local-path.ts#L20-L23

**Review by styfle — APPROVED** [review]

**Review by AndyBitz — APPROVED** [review]

**Comment by janzheng** [comment]
btw this error gets thrown when there's only a `now.json` file and no `vercel.json` file

**Pull request merged**